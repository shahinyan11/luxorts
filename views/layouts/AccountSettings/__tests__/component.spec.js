import { shallow } from 'enzyme';

import AccountSettingsLayout from 'views/layouts/AccountSettings';

describe('AccountSettingsLayout component', () => {
  const component = shallow(<AccountSettingsLayout>Foo</AccountSettingsLayout>);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
