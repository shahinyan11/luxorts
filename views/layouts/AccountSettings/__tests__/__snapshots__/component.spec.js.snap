// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountSettingsLayout component matches snapshot 1`] = `
<div
  className="main-layout"
>
  <Header
    isGuest={false}
    isSignIn={false}
    isSignUp={false}
    isWithFilter={false}
  />
  <div
    className="main-layout__content"
  >
    Foo
  </div>
  <Footer
    isAccount={true}
    isFooterMenu={false}
    isLanding={false}
    isRegistration={false}
  />
</div>
`;
